subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -644,6 +653,17 @@ static inline struct net_buf *process_node(struct node_rx_pdu *node_rx)
 	uint8_t class = node_rx->hdr.user_meta;
 	struct net_buf *buf = NULL;
 
+#if !defined(CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE)
+	if (class == HCI_CLASS_EVT_CMD_STATUS) {
+		struct node_rx_hci_evt *node_rx_hci = (void *)node_rx;
+
+		buf = node_rx_hci->buf;
+		node_rx_hci->buf = NULL;
+
+		return buf;
+	}
+#endif /* !CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE */
```

Comment:
I think these blocks for `!defined(CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE)` for this purpose could use some comments stating the purpose and reasoning.